app/graphql/join.ts

```diff
@@ -235,4 +234,5 @@ export const mergeObservations = (
 
   // Extract observations from the merged object indexed by joinBy value
   return Object.values(merged);
-};
+}; /** cubeIri to dimensionIds */
+export type JoinBy = Record<string, ComponentId[]>;
```

Comment:
Nit: Is it versioned or unversioned cube iri in the end? 😄 Maybe there could be a hint here? 🤔

Reply:
Thanks 🙇‍♂️

Reply:
It's clarified in the name of the type now in 217aacb